src/ops/ops.go

```diff
@@ -419,3 +427,35 @@ func (o *ops) ReloadHostFile(filepath string) error {
 	}
 	return nil
 }
+
+func (o *ops) ReadFile(filename string) ([]byte, error) {
+	return ioutil.ReadFile(filename)
+}
+
+func (o *ops) WriteFile(filename string, data []byte, perm os.FileMode) error {
+	return ioutil.WriteFile(filename, data, perm)
+}
```

Comment:
Is this really the only way to be able to test this?
It seems a shame to add unnecessary methods like this because of a test framework limitation.

Reply:
Obsolete